Documentation/cli-installation-scenarios.md

```diff
@@ -242,8 +262,16 @@ OSX/Linux:
 
 #### Windows obtain one-liner example 
 
+CMD:
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex}"
+```
+
+PowerShell:
+
 ```
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1'))}"
+iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex
```

Comment:
Looks great! Incidentally, if `BitsTransfer` is available on Windows Nano Server, can we try something like https://github.com/dotnet/cli/pull/4087#issuecomment-242447079 (the last code block). Followed by `rm ./dotnet-install.ps1`. IMO, that would simplify/cleanup the download function as we won't be needing S.I.F.Create() and S.N.H.